fixtures/dev-env/tests/index.test.ts

```diff
@@ -649,3 +649,42 @@ describe("startDevWorker: ProxyController", () => {
 		await expect(res.text()).resolves.toBe("UserWorker:3");
 	});
 });
+
+describe("regressions", () => {
+	// https://github.com/cloudflare/workers-sdk/issues/5095
+	test("#5095 requests where body (of >= 2 bytes) is not drained sometimes fail", async () => {
+		const run = await fakeStartUserWorker({
+			script: `
+				export default {
+					async fetch(request) {
+						return new Response("UserWorker:1");
+					}
+				}
+			`,
+		});
+
+		const COUNT = 30;
```

Comment:
This feels like it should be higher—are we guaranteed to see the failure case within 30 iterations?

Reply:
I was seeing something like 30% failures so 30 felt like enough. But I wasn't done tuning the value anyway. I'm still investigating (should've opened this PR as a draft 🤦)